public/503.html

```diff
@@ -92,11 +92,11 @@ <h3 class="landmark heading">Site Navigation</h3>
       <div class="clear"></div>
     </div>
     <!-- END header -->
-    <div id="main" class="system error 503 maintenance">
-      <h2>Error 503 - Service unavailable</h2>
-      <h3>Exceeded maximum allowed downloads per minute.</h3>
-      <p>To combat bots, we are currently banning IP addresses that download too many works in a short time period. If you see this page repeatedly please pause at least 10 seconds between downloads. If you are banned, you will be unable to access the Archive. Access will be restored 24 hours after the ban started.</p>
-      <p>Follow <a href="http://twitter.com/ao3_status/">@AO3_Status</a> on twitter for updates.</p>
+    <div id="main" class="system error 503">
+      <h2>Error 503</h2>
+      <h3>The page was responding too slowly.</h3>
+      <!--plain english suggestion: There are so many people using the archive right now, we can't show your page. -->
+      <p>Follow <a href="http://twitter.com/ao3_status/">@AO3_Status</a> on twitter for updates if this keeps happening.</p>
```

Comment:
Yes because you asked me to 

"Can we change this to just: <p>Follow <a href="http://twitter.com/ao3_status/">@AO3_Status</a> on Twitter for updates.</p>?"

I will update to upper case T every where.